src/pages/Home.js

```diff
@@ -1,13 +1,30 @@
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
 import Players from '../components/Players'
 import { Button } from '@mui/material'
 import { useNavigate } from 'react-router-dom'
 import { ChangeUsernameButton } from '../components/ChangeUsernameButton'
+import { updateHeartbeat } from '../api'
+import { userHeartbeatInterval } from '../constants'
+import { getLocalUser, localUserExists } from '../localStore'
 
-function Home() {
+export default function Home() {
   const navigate = useNavigate()
   const [username, setUsername] = useState('')
 
+  const updateUserHeartbeat = () => {
+    if (localUserExists()) {
+      const user = getLocalUser()
+      updateHeartbeat(user).catch((e) => console.log(e))
+    }
+  }
+
+  useEffect(() => {
```

Comment:
With the current implementation, when we open the page and our user is persisted in `localstorage` the app waits 15 seconds before it sends a heartbeat. This means that any new player would wait 15 seconds for their name to appear in the list. We need to send a heartbeat immediately after a local user is detected, and we need to do this initial heartbeat once.